src/main/java/uk/gov/hmcts/cmc/claimstore/services/ServiceAuthProviderService.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.cmc.claimstore.services;
+
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+
+public class ServiceAuthProviderService {
+    private final AuthTokenGenerator cachedServiceAuthTokenGenerator;
+
+    public ServiceAuthProviderService(final AuthTokenGenerator cachedServiceAuthTokenGenerator) {
```

Comment:
Any reason you created service? That does not give any value and makes traversing code more difficult. We need to stop creating useless classes.